### ISSUE TITLE
Create step by step doc on how to install EasyEngine on mac 

### ISSUE THREAD

**Issue by kirtangajjar** [EasyEngine/docs#47]
Seems like a good idea - https://community.easyengine.io/t/ee-v4-local-dev/11480

**Comment by pserafini** [comment]
yeah !

**Comment by pserafini** [comment]
Would be amazing to explain the the sync methodology between local and remote too ! 
That will be a fantastic resource for users.

**Comment by jasperf** [comment]
Yeah it would be. Working on setting up the same `domain.test` now and post nuking and starting I am having issues reaching the site. This howto should also mention not to use SSL as you are using localhost. That and some troubleshooting techniques.

**Comment by githubr** [comment]
Any progress on this? I just installed ee v4 on my local dev server and try to set up a site using local domain eg myproject.local and I got
> Warning: Problem connecting to site!
> Warning: Initiating clean-up

**Comment by kirtangajjar** [comment]
@githubr Your issue seems unrelated to this one. I request you to open a thread in community forums about it. Also, it would be very helpful if you could re-run the command with `--debug` flag (i.e. `ee site create myproject.local  --debug`) and paste it's entire output in the community thread.

**Comment by kalinstanchev** [comment]
Hello, any update on this? :)
